### PR TITLE
Add Academic Year to Student

### LINKED ISSUE #24: Add Academic Year field to Student

### LINKED ISSUE #25: Add academic year field to student properties
Add student property called academic year.

### PR DESCRIPTION
**Pull request by Whleee**

### CODE REVIEW

File: src/main/java/seedu/address/model/student/AcademicYear.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's name in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAcademicYear(String)}
+ */
+public class AcademicYear {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "The Academic Year should only contain a number between 1 to 5.";
```
Comment (faqihisaninja) [inline src/main/java/seedu/address/model/student/AcademicYear.java:13]: Isn't this supposed to be 1 to 6?

File: LICENSE [hunk 2]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any student obtaining a copy
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (faqihisaninja) [inline LICENSE:5]: You need to update to student for this file.

**Review by faqihisaninja — COMMENTED** [review]
Looks good man! See what the others say about it.

File: LICENSE (hunk 2 above)
Comment (sudogene) [reply LICENSE:5]: Ah that one is LICENSE, should stay as person

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 3]
```diff
@@ -79,6 +80,21 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String academicYear} into an {@code AcademicYear}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code email} is invalid.
```
Comment (sudogene) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: @throws ParseException if the given {@code **academicYear**} is invalid.

**Review by sudogene — COMMENTED** [review]
Looks good! Just that one minor documentation issue.

File: src/main/java/seedu/address/model/student/AcademicYear.java (hunk 1 above)
Comment (Whleee) [reply src/main/java/seedu/address/model/student/AcademicYear.java:13]: I checked the google docs and it said 1 to 5? hmm can someone confirm

Comment (sudogene) [reply src/main/java/seedu/address/model/student/AcademicYear.java:13]: @yejiadong @kerkpy

**Pull request merged**